build-docker.sh

```diff
@@ -7,45 +7,66 @@
 set -euo pipefail
 
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
-
 cd "${SCRIPT_DIR}"
-
 ROOT="$(pwd)"
+
 source "${ROOT}/DEPENDENCIES"
 
 GIT_REVISION=$(git rev-parse --short HEAD)
 git diff-files --quiet || GIT_REVISION="${GIT_REVISION}-dirty"
-
 GEDS_VERSION=${TRAVIS_TAG:-$(git describe --tags --match "v*" --dirty)}
 [[ "$GEDS_VERSION" =~ ^v ]] && GEDS_VERSION=$(echo $GEDS_VERSION | cut -c 2-)
 
-IMAGE_NAME=geds-dev_${DOCKER_BUILD_TARGET}
-IMAGE=us.icr.io/zrlio/${IMAGE_NAME}
+DOCKER_BUILD_TARGET="debian11.5"
+REGISTRY="${REGISTRY:-zac32.zurich.ibm.com}"
+PREFIX="${PREFIX:-${USER}}"
+DOCKER_VERSION=latest
+IMAGE_NAME="${REGISTRY}/${PREFIX}/geds-dev_${DOCKER_BUILD_TARGET}"
 
 BASE_BUILD_IMAGE="geds_build:${DOCKER_BUILD_TARGET}"
 docker build -t ${BASE_BUILD_IMAGE} \
     -f docker/Dockerfile-base_${DOCKER_BUILD_TARGET} .
 
-GRPC_DOCKER_IMAGE="geds_dependencies-${DOCKER_BUILD_TARGET}:${GIT_REVISION}"
-docker build -t ${GRPC_DOCKER_IMAGE} \
-    --build-arg DOCKER_BUILD_TARGET=${DOCKER_BUILD_TARGET} \
-    --build-arg GRPC_VERSION=${GRPC_VERSION} \
-    --build-arg AWS_SDK_VERSION=${AWS_SDK_VERSION} \
-    --build-arg BOOST_VERSION=${BOOST_VERSION} \
-    --build-arg BOOST_VERSION_=${BOOST_VERSION//./_} \
-    -f docker/Dockerfile-dependencies .
+GRPC_DOCKER_IMAGE="geds_dependencies-${DOCKER_BUILD_TARGET}:${DOCKER_VERSION}"
+# comment which lines : all
+#docker build -t ${GRPC_DOCKER_IMAGE} \
+#    --build-arg CMAKE_BUILD_PARALLEL_LEVEL=$(($(nproc) + 1)) \
+#    --build-arg DOCKER_BUILD_TARGET=${DOCKER_BUILD_TARGET} \
+#    --build-arg GRPC_VERSION=${GRPC_VERSION} \
+#    --build-arg AWS_SDK_VERSION=${AWS_SDK_VERSION} \
+#    --build-arg BOOST_VERSION=${BOOST_VERSION} \
+#    --build-arg BOOST_VERSION_=${BOOST_VERSION//./_} \
+#    -f docker/Dockerfile-dependencies .
```

Comment:
Revert.

Reply:
nvm. keep it for now. We'll have to think about building docker containers in the future.